packages/frontend/@n8n/design-system/src/components/N8nIcon/icons.ts

```diff
@@ -435,6 +436,7 @@ export const updatedIconSet = {
 	'node-dirty': NodeDirty,
 	'node-ellipsis': NodeEllipsis,
 	'node-error': NodeError,
```

Comment:
Can we rename this to `node-execution-error` so people don't misuse this for validation errors?

Reply:
Makes sense, updated